Sources/COW/COW.swift

```diff
@@ -248,3 +248,17 @@ extension _Box: Codable where Contents: Codable {
   }
   
 }
+
+// Idea of workaround credits to 
+// https://github.com/JosephDuffy/HashableMacro/commit/250787664a63ceff83c1c9b5e30e574ada568f2f.
```

Comment:
This page is not found. Could you find a permanent link or just use the GitHub repo's URL?

Reply:
I can open it without issue. Did you accidentally copy the trailing dot? The permalink works with the same idea behind: to access with commit hash. In case that it didn't work, it must have been the author intentionally destroy the commit history.

https://github.com/JosephDuffy/HashableMacro/blob/250787664a63ceff83c1c9b5e30e574ada568f2f/Sources/CustomHashable/CustomEqualityProviding.swift#L1

Reply:
Sorry. The link works without the trailing dot. Xcode automatically counts the trailing dot as a part of the URL. It could help Xcode disambiguate this if you remove that dot.